### DISCUSSION TITLE
vips8.12.2 error when parsing ndpi file format

### DISCUSSION THREAD

**Discussion by yzawudi** [libvips/libvips#3723] [Q&A]
## Questions, enhancements, tips, etc.
vips8.12.2 error when parsing ndpi file format

## Bug report
<img width="1019" alt="image" src="https://github.com/libvips/libvips/assets/42707709/296e5452-ea2c-46d9-abe7-e16083505330">**Describe the bug**

**Environment**
(please complete the following information)
 - OS: Centos7.4
 - Vips: 8.12.2
Do I need to prepare the input image in a specific way?
Thanks for help,
Thomas

**Comment by yzawudi** [comment]
Using vips dzsave on the ndpi file downloaded from the openslide official website will cause an error and kill the command

**Comment by jcupitt** [comment]
Hello @yzawudi,

Your libvips has been built without openslide support. You'll need to look into where the binary came from, and if they have a version which includes this feature.

**Reply by yzawudi** [reply]
OK, thanks, I will try to add openslide
